hazelcast/src/main/java/com/hazelcast/core/HazelcastInstance.java

```diff
@@ -416,6 +416,8 @@
      */
     ICacheManager getCacheManager();
 
+    ICardinalityEstimator getCardinalityEstimator(String name);
```

Comment:
this should have a really good JavaDoc. This is the place where you need to "sell" the data structure! :)